README.md

```diff
@@ -71,83 +89,130 @@ Generic resourceId 0x00000000000000000000000000000000000000000000000000000000000
 ===============================================
 ```
 
-This means that you have all the address that you need to run the UI locally.
-
-A quick note aside: if you want to check the logs of your nodes or the relayers, you can go `/e2e/evm-evm` folder and run the following command:
+It means you have all the addresses to run the UI locally.
+**A quick note aside:** if you want to check the logs of your nodes or the relayers, you can go to `/e2e/evm-evm` folder
+and
+run the following command:
 
 ```bash
 # inside the root directory of sygma
 cd example
 docker-compose -f ./docker-compose.yml logs setup
 ```
 
-This is going to output the `relayer1` logs. You can also run the command with the `-f` flag to follow the output of your services. If you want to see all the logs of your services just run `docker-compose -f docker-compose.yml logs -f`.
+It will output the `relayer1` logs. You can also run the command with the `-f` flag to follow the output of your
+services. To see all the logs of your services run
+
+```bash
+docker-compose -f docker-compose.yml logs -f
+```
+
+&nbsp;
 
-After you get the address for the contracts deployed on your local setup, we need to add this to the `runtime` config of our UI.
+### Sygma UI setup
 
-Go to `/packages/example` that contains our full UI, and inside the config folder, edit the `sygma-runtime-config.evm.js` file with the addresses that you got after the deploy.
+After you get the address for the contracts deployed on your local setup, we need to add this to the `runtime` config
+of our UI.
+
+Go to `/packages/sygma-ui/public` and inside the folder, create the `sygma-runtime-config.json` file with the addresses
+that you got after the deployment of Sygma project.
 
 You will end up with something like this:
 
-```js
-window.__RUNTIME_CONFIG__ = {
-  INDEXER__URL: "http://localhost:8000",
-  SYGMA: {
-    chains: [
+```json
+{
+  "UI": {
+    "transactionAutoUpdateInterval": 5000,
+    "nftTokenPage": true
+  },
+  "SYGMA": {
```

Comment:
please add latest config here and further:
````
  {
    domainId: "1",
    networkId: 1337,
    name: "Local EVM 1",
    decimals: 18,
    bridgeAddress: "0x6CdE2Cd82a4F8B74693Ff5e194c19CA08c2d1c68",
    erc20HandlerAddress: "0x02091EefF969b33A5CE8A729DaE325879bf76f90",
    erc721HandlerAddress: "0xC2D334e2f27A9dB2Ed8C4561De86C1A00EBf6760",
    rpcUrl: "http://localhost:8545",
    tokens: [
      {
        type: "erc20",
        address: "0x78E5b9cEC9aEA29071f070C8cC561F692B3511A6",
        name: "ERC20LRTST",
        symbol: "ETHIcon",
        imageUri: "ETHIcon",
        decimals: 18,
        resourceId:
          "0x0000000000000000000000000000000000000000000000000000000000000300",
        feeSettings: {
          type: "basic",
          address: "0x8dA96a8C2b2d3e5ae7e668d0C94393aa8D5D3B94",
        },
      },
    ],
  },
  {
    domainId: "2",
    networkId: 1338,
    name: "Local EVM 2",
    decimals: 18,
    bridgeAddress: "0x6CdE2Cd82a4F8B74693Ff5e194c19CA08c2d1c68",
    erc20HandlerAddress: "0x02091EefF969b33A5CE8A729DaE325879bf76f90",
    erc721HandlerAddress: "0x481f97f9C82a971B3844a422936a4d3c4082bF84",
    rpcUrl: "http://localhost:8547",
    tokens: [
      {
        type: "erc20",
        address: "0x78E5b9cEC9aEA29071f070C8cC561F692B3511A6",
        name: "ERC20LRTST",
        symbol: "ETHIcon",
        imageUri: "ETHIcon",
        decimals: 18,
        resourceId:
          "0x0000000000000000000000000000000000000000000000000000000000000300",
        feeSettings: {
          type: "basic",
          address: "0x8dA96a8C2b2d3e5ae7e668d0C94393aa8D5D3B94",
        },
      },
    ],
  },

Reply:
The config you provided is from the actual dev environment, is it correct?
I think that as you follow the steps in the readme, in the previous section you run the sygma locally and get the environment variables:
```text
- Chain 1 -
Bridge: 0xd606A00c1A39dA53EA7Bb3Ab570BBE40b156EB66
ERC20: 0x75dF75bcdCa8eA2360c562b4aaDBAF3dfAf5b19b
ERC20 Handler: 0xb83065680e6AEc805774d8545516dF4e936F0dC0
ERC721: 0xb911DF90bCccd3D76a1d8f5fDcd32471e28Cc2c1
ERC721 Handler: 0x05C5AFACf64A6082D4933752FfB447AED63581b1
Generic Handler: 0x7573B1c6de00a73e98CDac5Cd2c4a252BdC87600
Asset Store: 0x3cA3808176Ad060Ad80c4e08F30d85973Ef1d99e
ERC20 resourceId: 0x0000000000000000000000000000000000000000000000000000000000000000
ERC721 resourceId 0x0000000000000000000000000000000000000000000000000000000000000200
Generic resourceId 0x0000000000000000000000000000000000000000000000000000000000000100
```
So in the current step, we put the values from the local config to 'sygma-runtime-config.json'. It's better to keep the reference of the test environment as you can compare where the data is coming from. A user might be confused about why he/she gets the variables in the previous step and then uses a different config.

Reply:
Yes, as an example, makes sense.